From 60e5e4b3bc8eccbcda9511fea7141963030d5af7 Mon Sep 17 00:00:00 2001
From: Liao Pingfang <liao.pingfang@zte.com.cn>
Date: Tue, 14 Jul 2020 18:59:12 +0800
Subject: [PATCH 067/197] cifs: Remove the superfluous break

Remove the superfuous break, as there is a 'return' before it.

Signed-off-by: Liao Pingfang <liao.pingfang@zte.com.cn>
Signed-off-by: Yi Wang <wang.yi59@zte.com.cn>
Signed-off-by: Steve French <stfrench@microsoft.com>
---
 fs/cifs/sess.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/cifs/sess.c b/fs/cifs/sess.c
index 5d05bd2822d2..6708ab0aad1c 100644
--- a/fs/cifs/sess.c
+++ b/fs/cifs/sess.c
@@ -1705,7 +1705,6 @@ static int select_sec(struct cifs_ses *ses, struct sess_data *sess_data)
 #else
 		cifs_dbg(VFS, "Kerberos negotiated but upcall support disabled!\n");
 		return -ENOSYS;
-		break;
 #endif /* CONFIG_CIFS_UPCALL */
 	case RawNTLMSSP:
 		sess_data->func = sess_auth_rawntlmssp_negotiate;
-- 
2.27.0

